packages/@aws-cdk/aws-kinesisfirehose-alpha/test/integ.delivery-stream.source-stream.js.snapshot/aws-cdk-firehose-delivery-stream-source-stream.template.json

```diff
@@ -189,6 +189,9 @@
   "DeliveryStreamNoSourceOrEncryptionKey0E4AAB82": {
    "Type": "AWS::KinesisFirehose::DeliveryStream",
    "Properties": {
+    "DeliveryStreamEncryptionConfigurationInput": {
+     "KeyType": "AWS_OWNED_CMK"
+    },
```

Comment:
Didn't we use `unencrypted` in the integration test file?

Reply:
oh wait you're right. that's weird..lemme double check that